packages/ui/src/components/filters/filters-bar/filters-bar.tsx

```diff
@@ -114,17 +161,19 @@ const FiltersBar = ({
             </button>
           </div>
 
-          <Views
-            currentView={viewManagement.currentView}
-            savedViews={viewManagement.savedViews}
-            viewManagement={{
-              ...viewManagement,
-              activeFilters,
-              activeSorts,
-              saveView: (name: string) => viewManagement.saveView(name, activeFilters, activeSorts)
-            }}
-            hasChanges={!!hasViewChanges}
-          />
+          {viewManagement && (
+            <Views
+              currentView={viewManagement.currentView}
+              savedViews={viewManagement.savedViews}
+              viewManagement={{
+                ...viewManagement,
+                activeFilters,
+                activeSorts,
+                saveView: (name: string) => viewManagement.saveView(name, activeFilters, activeSorts)
+              }}
+              hasChanges={!!hasViewChanges}
```

Comment:
Might as well simplify a bit and _not_ extract `hasViewChanges` as a variable above and just use it here. There'll be no change in performance, but will mean you can remove optional chaining.
```suggestion
              hasChanges={viewManagement.hasActiveViewChanges(activeFilters, activeSorts)}
```

Or, if this function is expensive (or has the potential to be expensive), wrap the call on line 48 with a `useMemo`.